libs/openFrameworks/3d/ofEasyCam.cpp

```diff
@@ -132,33 +148,32 @@ char ofEasyCam::getTranslationKey() const{
 }
 
 //----------------------------------------
-void ofEasyCam::enableMouseInput(){
-	if(!bMouseInputEnabled && events){
-		listeners.push_back(events->update.newListener(this, &ofEasyCam::update));
-		listeners.push_back(events->mouseDragged.newListener(this, &ofEasyCam::mouseDragged));
-		listeners.push_back(events->mousePressed.newListener(this, &ofEasyCam::mousePressed));
-		listeners.push_back(events->mouseReleased.newListener(this, &ofEasyCam::mouseReleased));
-		listeners.push_back(events->mouseScrolled.newListener(this, &ofEasyCam::mouseScrolled));
+void ofEasyCam::enableMouseInput(bool e){
```

Comment:
I just had it like this and I like the enable(bool) pattern. But right, if it is not used anywhere else I can remove it.